animations-dbus/animations-dbus-errors.c

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2018 Endless Mobile, Inc.
+ *
+ * animations-dbus is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 2.1 of the
+ * License, or (at your option) any later version.
+ *
+ * libanimation-dbus is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with libanimation-dbus.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
```

Comment:
Add yourself as the author? Credit where credit is due.
And the same in other occasions.

Reply:
;-)